go/oasis-node/cmd/debug/txsource/workload/workload.go

```diff
@@ -92,6 +94,15 @@ func (bw *BaseWorkload) GasPrice() uint64 {
 	return gasPrice.ToBigInt().Uint64()
 }
 
+// LoadNonce returns the current nonce for the provided address.
+func (bw *BaseWorkload) LoadNonce(ctx context.Context, addr staking.Address) (uint64, error) {
```

Comment:
Note that there's also the `GetSignerNonce(ctx context.Context, req *GetSignerNonceRequest) (uint64, error)` method on the consensus backend which gets you the same thing so maybe you should use this instead of this method?

Reply:
thanks for noticing! as you mentioned "offline" loading the nonce was indeed not needed :-)